migrations/alembic.ini

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# A generic, single database configuration.
+
+[alembic]
+# template used to generate migration files
+# file_template = %%(rev)s_%%(slug)s
+
+# set to 'true' to run the environment during
+# the 'revision' command, regardless of autogenerate
+# revision_environment = false
```

Comment:
Should these be removed/uncommented?

Reply:
This file is itself autogenerated as well, a one-time deal. I don't see a harm in removing or keeping these comments, but there is no need to uncomment them. I will remove them to keep this file clean.